dts/msm8226/msm8926-samsung-r04.dts

```diff
@@ -0,0 +1,21 @@
+// SPDX-License-Identifier: GPL-2.0-only
+
+/dts-v1/;
+
+#include <skeleton.dtsi>
+#include <lk2nd.h>
+
+/ {
+	// This is used by the bootloader to find the correct DTB
+        qcom,msm-id =  <0x6E08FF01 4 0x20000>;
```

Comment:
https://github.com/msm8916-mainline/lk2nd/blob/experimental-tmp2/lk2nd/include/dt-bindings/arm/qcom%2Cids.h#L90

Where do you know about `0x6E` (110)?

Reply:
i renamed dts